shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/files.py

```diff
@@ -91,14 +89,18 @@ def combine_folder_and_file_name(folder_name: str, file_name: str) -> str:
     Returns:
     str: The combined folder and file name as a single path.
     """
-    return os.path.join(folder_name, file_name)
+
+    combined_name = os.path.normpath(
```

Comment:
Why was this changed?

Reply:
join did not handle the leading `/` the same as normpath

Reply:
figured more details would probably be helpful
file_name =`/test.txt` 
folder_name = '/parent_folder/'

os.path.join(folder_name,file_name) outputs /test.txt
os.path.normpath(...) outputs parent_folder/test.txt

if the file name didn not have the leading `/` it would come out to the same value